packages/fx-core/src/plugins/solution/fx-solution/executor.ts

```diff
@@ -118,7 +120,7 @@ export async function executeLifecycles(
   lifecycles: LifecyclesWithContext[],
   postLifecycles: LifecyclesWithContext[],
   onPreLifecycleFinished?: () => Promise<Result<any, FxError>>,
-  onLifecycleFinished?: () => Promise<Result<any, FxError>>,
+  onLifecycleFinished?: (result?: any[]) => Promise<Result<any, FxError>>,
```

Comment:
How about adding result to each `onXXXFinshed()`'s parameter for consistency. And reorder parameters to preLifecycles, onPreLifecycleFinished, lifecycles, onLifecycleFinished, postLifecycles, onPostLifecycleFinished

Reply:
As onXXXFinished are optional, I will not change the order. But I can add parameters for them.